src/Sylius/Behat/Context/Setup/CustomerContext.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Context\Setup;
+
+use Behat\Behat\Context\Context;
+use Sylius\Component\Core\Model\CustomerInterface;
+use Sylius\Component\Core\Test\Services\SharedStorageInterface;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+*/
+class CustomerContext implements Context
```

Comment:
I would suggest to leave them separated, because usually it is enough to operate on customer only, so we do not need logic from UserContext.